apps/aering/test/aer_abi_tests.erl

```diff
@@ -0,0 +1,12 @@
+-module(aer_abi_tests).
+
+-include_lib("eunit/include/eunit.hrl").
+
+encode_call_with_integer_test() -> 
+    <<0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
```

Comment:
I understand spec is https://github.com/aeternity/protocol/pull/9/files#diff-a7778cdeae90e4a2ccdc834c003760e7R227

Reply:
The (offline) spec has been updated as the language was implemented. I have to fix the protocoll repo.